website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -235,13 +235,117 @@ This property contains the details of the row that triggered the action (`onSubm
+
+Imagine that your table gets information from a query called `myAPI_get`, which is a GET request to a datasource called `myAPI`.
+
+```javascript
+// In the Table Data field
+{{myAPI.data}}
+
+// Returns [{fruit: "pomegranate", color: "red"}, ... ]
+```
+
+And perhaps you add a new row to the table with the data:
+
+```javascript
+{fruit: "banana", color: "yellow"}
+```
+
+To update the original datasource with the new data, you might execute a query like `myAPI_post`, which would be a POST request to `myAPI`. To add the data to the POST request, you can use the Table's `newRow` attribute to reference the new row.
+
+```javascript
+// In our `myAPI_post` query:
+{{Table1.newRow}}
+
+// In the Table's `onSave` event:
+{{myAPI_post.run()}}
+```
+
+Assuming that the POST request is successful, the `myAPI` datasource receives the new information, and the Table should contain the new row the next time it updates.
+
+### Properties
+
+The following properties are related to adding new rows:
+
+| Property              | Type        | Definition               | Code Snippet        |
+| --------------------- | ----------- | ------------------------ | ------------------- |
+| **Allow adding a row** | Widget | Toggles a button in the table which allows users to submit new rows of data. Only columns marked as **Editable** can accept user input. Use the **onSave** event to update the source of the table's data and reflect the user's changes. |
+| **Default Values** | Widget | The values to automatically populate the new row with when a user begins creating a new row. Expects an object with the same keys as the columns in the existing table data. |
+| **isAddRowInProgress** | Binding | Indicates whether a new row is currently being added by the user. | `Table1.isAddRowInProgress` |
+| **newRow** | Binding | When writing code for validation properties, this variable contains a reference to the new row object added by the user. | `Table1.newRow` |
+| **isNewRow** | Validation | When writing code for validation properties, this variable indicates whether the cells being validated are part of a new or an existing row. | `isNewRow` |
+
+The subheadings below describe these properties in more detail:
+
+#### Allow adding a row
+
+Turning this on allows users to click a button on the table to start a new row in the table. Users can only input data for columns that are marked as **Editable** in the table properties.
+
+Use the **onSave** event to update the source of the table's data and reflect the user's changes. For example, the **onSave** event might be set to trigger a query that sends the data from a new table row to the database that supplies the table with its values.
+
+#### Default Values
+
+<VideoEmbed host="youtube" videoId="3KozovP7LNk" title="Table | Add new rows | Default values" caption="Set default values for new Table rows"/>
```

Comment:
Throughout this page, i noticed the captions below the videos don't stand out as captions. Let's use the standard as on the other pages

Reply:
I'm not sure what you mean here, on my end all of the captions on every video I've seen appear as plain text. Which is odd of course -- but they do appear this way consistently for me.